WhoIsThatServer.Storage/Controllers/FaceFeaturesController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Http;
+using WhoIsThatServer.Storage.ErrorMessages;
+using WhoIsThatServer.Storage.Exceptions;
+using WhoIsThatServer.Storage.Helpers;
+using WhoIsThatServer.Storage.Models;
+
+namespace WhoIsThatServer.Storage.Controllers
+{
+    public class FaceFeaturesController : ApiController, IFaceFeaturesController
+    {
+        public IFaceFeaturesHelper FaceFeaturesHelper { get; set; } = new FaceFeaturesHelper();
+
+        /// <inheritdoc/>
+        [HttpPost]
+        [Route("api/features/add")]
```

Comment:
Remember our conversation about HTTP methods and verbs in the route ;)

Reply:
damn me, of course I remember :D

Reply:
Fixed it for now. You just made me realize that I completely forgot to oversee my route naming. It's in my "Todo bucket" right now :)